tools/odbc/test/CMakeLists.txt

```diff
@@ -25,4 +25,10 @@ add_executable(
   tests/quotes.cpp
   tests/result_conversion.cpp)
 
+if(CMAKE_GENERATOR MATCHES "Visual Studio")
```

Comment:
Only comment, I find a bit unclear the branch on CMAKE_GENERATOR, I think it would be better to have some external variable like ODBC_EXTENDED_TESTS, set by default to 1 on Windows and 0 otherwise, but allowing that to be overridden, and here erroring out if that's needed and CMAKE_GENERATOR is not "Visual Studio".

Or keeping changes minimal, adding a warning (not an error) informing that this build step has been skipped.

Reply:
Since this is C++ CLI it uses COMMON_LANGUAGE_RUNTIME property which only works when CMAKE_GENERATOR is Visual Studio. It does not work with Ninja or nmake. Figured it was best to be explicit here...